### DISCUSSION TITLE
My verification gets rejected every time

### DISCUSSION THREAD

**Discussion by jachymg** [community/community#139646] [GitHub Education] [answered]
### Hi everyone,

I'm trying to get verified as a student for the Education pack, but I get rejected every time. I sent them a photo of proof of status student, which is a official stamped document by my school however I got rejected. I even uploaded a version where I translated it, but still got rejected. What am I doing wrong? Is there any way to contact Github staff/helpdesk directly?

Edit: also good thing to mention is that I registered this account with my school email and have entered my details in the billing information

Thanks in advance

**Comment by jachymg** [comment, marked as answer]
Update, it got approved, I tried to make a photo using a phone instead of PC and it worked

**Reply by anil826** [reply]
For me, it's the same issue I have a valid document but got rejected even when tried via Phone but it looks like in the IOS browser, the back camera is not working/supported and the front camera is not stable enough to take images of documents.

Any help would be appreciated

**Reply by jachymg** [reply]
I used phone with Google Chrome and it for some reason worked on the first try :D
